docs/balena-cli.md

```diff
@@ -333,6 +333,30 @@ Examples:
 
 ### Options
 
+#### --fields FIELDS
```

Comment:
Maybe we need to split these in separate commits to communicate to users what changed (rather than the not-that-useful "Cleanup all isV14 switches"). I'm guessing that using "blame" on GH to find the PR that added those version checks should help find a better commit message.